en/docs/manage-databases-and-caches/add-choreo-managed-databases-and-caches-to-the-marketplace.md

```diff
@@ -20,8 +20,13 @@ To register credentials for a database, follow these steps:
 5. Click the **Databases** tab.
 6. Click to expand the database for which you want to register credentials, then click **Add Credentials**.
 7. In the **Add Credentials** dialog, do one of the following depending on your requirements:
-    - If you want to use the custom credentials you created using the super admin credentials, select **Add New Credentials** and specify appropriate values for each field.
-    - If you want to use the default super admin credentials, select **Add Super Admin Credentials** and specify appropriate values for each field.
+
+    i. **Add New Credentials**
```

Comment:
Let's avoid the numbering since we are asking users to select one of the two options.
Suggestion:

- Select **Add New Credentials** and specify the following:
         - A display name for the credentials.
         - The database credentials you created to use when setting up database connections in Choreo. 
         - The environment where you want to use the credentials. 
         
         The steps to create new user credentials depend on the type of database or cache you are using. In general, you must log in to your database server as the super admin using a command-line interface or a database management tool. Then, create new database users with the required permission levels.

    - Select **Add Super Admin Credentials** and specify the following:
         - A display name for the credentials.
         - The default super admin credentials to use when setting up database connections in Choreo. 
         - The environment where you want to use the credentials.

Reply:
The content will look like this if the suggested changes are applied:
<img width="1406" alt="Screenshot 2024-11-08 at 16 59 32" src="https://github.com/user-attachments/assets/6be09185-8dca-486b-b823-6d0925a34372">

I can apply the changes if the suggestions are good to go.

Reply:
Can we remove the second point in Add Super Admin Credentials
`The default super admin credentials to use when setting up database connections in Choreo.` as user does not require to provide it, values will be auto populated